backend/src/main/game/cards/playable_card_logic.py

```diff
@@ -0,0 +1,50 @@
+
+class PlayableCardLogic:
+    def __init__(self, hand):
+        self.hand = hand
+        self.discard = []
+        self.lost = []
+        if not self.hand:
+            raise ValueError('wrong input value')
+
+    def hand_to_discard(self, card_name):
+        for card in self.hand:
+            if card.name == card_name:
```

Comment:
there's a lot of code duplication in this file.

I would suggest adding two methods:
```
def does_card_have_name(card, name):
  return card.name == name
```
and another function
```
def move_card(card, source, dest):
  source.remove(card)
  dest.append(card)
```
If you do this each of your functions then get shortened to:
```
def hand_to_lost(self, card_name):
  for card in self.hand:
    if self.does_card_have_name(card, name):
      self.move_card(card, self.hand, self.lost)
      break
```
You can even further refactor by moving everything up one layer of abstraction:
```
def move_card_by_name(self, card_name, source, destination):
  for card in  source:
    if self.does_card_have_name(card, name):
      self.move_card(card, source, destination)
      break
```
then every one of these functions turn into
```
def hand_to_lost(self, card_name):
  self.move_card_by_name(card_name, self.hand, self.lost)
```